packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderContext/DataConfigs.js

```diff
@@ -99,6 +99,7 @@ export const operatorConfig = [
     type: 'date',
   },
 ];
+export const variants = ['Non-Hierarchical', 'Hierarchical'];
```

Comment:
What do you think about making each variant a constant instead of an array with the two? I'm just seeing some potential confusion if `variants[1]` is used accidentally instead of `variants[0]`.

Reply:
It would also be clearer which variant you're using, ie `variants[0]` vs `SENTENCE_VARIANT`.

Reply:
agreed. i would just export these as individual strings to make it more clear what the variable is.

Reply:
i was thinking , if they again change the variant names this would be easier.

Reply:
But for readability aspect I could export as individual strings.